README.MD

```diff
@@ -38,8 +41,9 @@ Give a ⭐️ if you like this project!
 ## Acknowledgments
 
 - Hat tip to anyone who's code/resource was used
-- Inspiration
-- etc
+- Image Credits: (https://www.istockphoto.com)
+- Cindy Shin: [@adgio07](https://www.behance.net/adagio07)
```

Comment:
- [ ] It is good that you acknowledged the Cindy. However, you can use `Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07)` instead to indicate that the original design idea is by Cindy.
![image](https://user-images.githubusercontent.com/44057863/169669504-01b0daa0-3e12-47b4-a0e8-48e0e4877242.png)
#### Appropriate attribution
![image](https://user-images.githubusercontent.com/44057863/169669495-f929f22c-086c-4e0a-9b70-0f93bc636aa1.png)